python/lib/core/dmod/core/common/helper_functions.py

```diff
@@ -23,8 +22,8 @@
 
 from .types import TypeDefinition
 
-_CLASS_TYPE = typing.TypeVar('_CLASS_TYPE')
-"""A type that points directly to a class. The _CLASS_TYPE of `6`, for example, is `<class 'int'>`"""
+_CLASSTYPE = typing.TypeVar('_CLASSTYPE')
```

Comment:
`_CLASSTYPE` is used instead of `_T` for basically the same season `_KT` instead of `_T` - `_T` generally equates to "any type of whatever" whereas `_CLASSTYPE` is supposed to hint as some time of class (`<class 'int'>` instead of `int`). I forgot what the rub was, but there was a situation or two where `typing.Type[_T]` ran afoul of some type hinting checks. If I remember right, the `_T` in `typing.Type[_T]` doesn't really matter where `_CLASSTYPE` is used.

Reply:
I think binding `_CLASSTYPE` to `type` actually fixes a lot of stuff. Imma look into that.